src/utils/setTranslationWithFallback.js

```diff
@@ -0,0 +1,13 @@
+import { flatten } from "./flatten";
+
+export const setTranslationWithFallbackField = (locale, obj, fallbackField, ...field) => {
+	if (!obj || !obj.getIn(flatten([field]))) return obj;
+	const value = obj.getIn(flatten([field, locale])) || obj.get(fallbackField) || "";
+	return obj.setIn(flatten([field]), value);
```

Comment:
looks like similar functions 

we can simplify this function
return setTranslationWithFallbackValue(locale, obj, obj?.get(fallbackField), ...field)

Reply:
done